tasks/prettier_conformance/prettier.snap.md

```diff
@@ -506,6 +506,9 @@ Compatibility: 109/601 (18.14%)
 ### nullish-coalescing
 * nullish-coalescing/nullish_coalesing_operator.js
 
+### object-colon-bug
+* object-colon-bug/bug.js
```

Comment:
This regressed because we don't consider the following case when printing objects.

If the current objec tlooks like this:
```
const foo = { bar...
```
then we try to go into 1x line
else:
```
const foo = {
   bar...
   ```
then we don't care about trying to fit it in a single line.

I think this is ok for now?

Reply:
Regressions are ok, gotta fix things one at a time 😁